website/docs/d/destination_of_redirected_traffic.html.markdown

```diff
@@ -24,16 +33,16 @@ data "aci_destination_of_redirected_traffic" "example" {
 
 ## Argument Reference
 
-- `service_redirect_policy_dn` - (Required) Distinguished name of parent Service Redirect Policy object.
+- `service_redirect_policy_dn` - (Required) Distinguished name of the parent Service Redirect Policy object.
 - `ip` - (Required) The IP address.
 
 ## Attribute Reference
 
 - `id` - Attribute id set to the Dn of the Destination of redirected traffic.
-- `annotation` - (Optional) Annotation for object destination of redirected traffic.
+- `annotation` - (Optional) Annotation for the object destination of redirected traffic.
 - `destination` - (Optional) Specifies the description of a policy component.
-- `dest_name` - (Optional) The destination name to which data was exported. This utility creates a summary report containing configuration information, logs and diagnostic data that will help TAC in troubleshooting and resolving a technical issue.
-- `ip2` - (Optional) IP2 for object destination of redirected traffic.
+- `dest_name` - (Optional) The destination name to which data was exported. 
+- `ip2` - (Optional) IP2 for the object destination of redirected traffic.
```

Comment:
IP2 for the object destination of redirected traffic. -> IP2 of the destination of redirected traffic object.